spec/offchain.tex

```diff
@@ -185,45 +189,75 @@ \subsubsection{Processing transactions off-chain}
+	transaction which inputs are added to the confirmed UTxO in the Head, we include
+  the inputs to the \emph{local} ledger state so that they become available.
+  Finally the corresponding pending increment
+  transaction is cleared and the observed version is used for future snapshots by setting
+  $\hatv = v$. The version of the open head state is incremented on each \mtxIncrement{}
+  transaction as described in Section~\ref{sec:increment-tx}
+}
+\todo{missing to mention the multicast of $\hpRS$} \\
 
 \dparagraph{$\hpRS$.}\quad Upon receiving request
-$(\hpRS,v,s,\mT_{\mathsf{req}}, \tx_{\omega})$\footnote{Snapshot requests
+$(\hpRS,v,s,\mT_{\mathsf{req}}, \red{\tx_\alpha}, \tx_\omega)$\footnote{Snapshot requests
 	with only transaction identifiers are possible too if all parties keep an
 	index of previously seen transactions and their identifiers.} from party
 $\party_j$, the receiving $\party_i$ $\Req$s that $v$ refers to the current
 open state version, $s$ is the next snapshot number
 and that party $\party_j$ is responsible for leading its creation.\todo{define
 	$\hpLdr$} Party $\party_i$ may has to $\KwWait$ until the previous snapshot is
-confirmed ($\bar{\mc S}.s = \hats$). If the decommit transaction $\tx_{\omega}$
-is not $\bot$, the transaction is $\Req$d to be applicable to the last confirmed
-UTxO set $\bar{\mc S}.U$ and decommitted transaction outputs must be removed, yielding the still
-active UTxO set $U_{\mathsf{active}}$. Then, all requested transactions
-$\mT_{\mathsf{req}}$ are $\Req$d to be applicable to $U_{\mathsf{active}}$,
-otherwise the snapshot is rejected as invalid. Only then, $\party_i$ increments
-their seen-snapshot counter $\hats$, resets the signature accumulator
-$\hatSigma$, and computes the UTxO set of the new local snapshot as
-$U \gets U_{\mathsf{active}} \applytx \mT_{\mathsf{req}}$. Then, $\party_i$
-creates a signature $\msSig_i$ using their signing key $\hydraSigningKey$ on a
-message comprised by the $\cid$, the new snapshot number $\hats$, the new $\eta$
-resulting from canonically combining $U$ (see Section~\ref{sec:close-tx} for
-details), and $\eta_{\omega}$ derived from decommit transaction $\tx_{\omega}$ outputs.
-The signature is sent to all head members via message $(\hpAS,\hats,\msSig_i)$.
-Finally, the local ledger state $\hatmL$ and pending transaction set $\hatmT$
-get pruned by re-applying all locally pending transactions $\hatmT$ to the just
-requested snapshot's UTxO set iteratively and
-ultimately yielding a ``pruned'' version of $\hatmT$ and $\hatmL$. \\
+  confirmed ($\bar{\mc S}.s = \hats$). Then either:
+\begin{itemize}
+  \item
+    there is a pending decrement and we check that $\tx_\omega$ transaction is applicable to the last confirmed
+	  UTxO set $\bar{\mc S}.U$ and it's outputs are removed, yielding the still active UTxO set $U_{\mathsf{active}}$
+\red{\item
+    there is a pending increment $\tx_\alpha$ which inputs will be made available in the active UTxO set
```

Comment:
Should: not add this here (and remove the itemization again). There is nothing done on `reqSn` related to the increment.